scilpy/io/utils.py

```diff
@@ -932,32 +1069,37 @@ def range_checker(arg: str):
     return range_checker
 
 
-def get_default_screenshotting_data(args):
-    volume_img = nib.load(args.in_volume)
+def get_default_screenshotting_data(args, peaks=True):
+    volume_img = nib.load(args.volume)
 
-    transparency_mask_img = None
-    if args.in_transparency_mask:
-        transparency_mask_img = nib.load(args.in_transparency_mask)
+    transparency_img = None
+    if args.transparency:
+        transparency_img = nib.load(args.transparency)
 
     labelmap_img = None
-    if args.in_labelmap:
-        labelmap_img = nib.load(args.in_labelmap)
+    if args.labelmap:
+        labelmap_img = nib.load(args.labelmap)
+
+    ovl_imgs, ovl_colors = None, None
+    if args.overlays:
+        ovl_imgs = [nib.load(f) for f in args.overlays]
 
-    mask_imgs, masks_colors = None, None
-    if args.in_masks:
-        mask_imgs = [nib.load(f) for f in args.in_masks]
+        if args.overlays_colors is not None:
+            if len(args.overlays_colors) == 3:
+                ovl_colors = np.repeat(
+                    [args.overlays_colors], len(args.overlays), axis=0)
+            elif len(args.overlays_colors) // 3 == len(args.overlays):
+                ovl_colors = np.array(args.overlays_colors).reshape((-1, 3))
 
-        if args.masks_colors is not None:
-            if len(args.masks_colors) == 3:
-                masks_colors = np.repeat(
-                    [args.masks_colors], len(args.in_masks), axis=0)
-            elif len(args.masks_colors) // 3 == len(args.in_masks):
-                masks_colors = np.array(args.masks_colors).reshape((-1, 3))
+            ovl_colors = ovl_colors / 255.
 
-            masks_colors = masks_colors / 255.
+    peaks_imgs = None
+    if peaks and args.peaks:
+        peaks_imgs = [nib.load(f) for f in args.peaks]
 
     return volume_img, \
```

Comment:
Could you please format on one line, or if too long, with parethesis: (volume_img, ..., peaks_img)